src/clims/api/endpoints/trigger_script.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from rest_framework.permissions import IsAuthenticated
+
+from sentry.api.base import DEFAULT_AUTHENTICATION
+from rest_framework.response import Response
+from rest_framework import status
+from sentry.api.bases.organization import OrganizationEndpoint
+from clims.api.serializers.script_trigger import ScriptTriggerSerializer
+
+
+class TriggerScriptEndpoint(OrganizationEndpoint):
+    authentication_classes = DEFAULT_AUTHENTICATION
+    permission_classes = (IsAuthenticated, )
+
+    def post(self, request, organization):
+        # This endpoint is constructed in order to trigger a plugin script,
+        # specified by input parameters.
+        serializer = ScriptTriggerSerializer(data=request.data)
```

Comment:
My idea was that the UI programmer can never trigger a script directly, so in that case an endpoint would not be required for this. Rather I envisioned it so that the custom scripts would always be triggered by some service class.

Example:
* User creates a workbatch via the WorkflowService
* The workflow service triggers the on_create handler

Reply:
I don't follow you here. I don't see how an api endpoint count NOT be triggered after a user buttonclick?

Reply:
Right, sorry for the confusion, we'll need something like this in the case of button clicks, but in the case of `on_create` I was thinking that it happens when the workbatch gets created.

About the name "trigger script". I wonder if we could have it something like "trigger action"?